samples/keypoint_detection/README.md

```diff
@@ -2,13 +2,15 @@
 
 This application demonstrates the human body key point detection model.
 
+The model is yolov8n-pose from [ultralytics](https://github.com/ultralytics/ultralytics). It is exported to ONNX using ultralytics cli `yolo export model=yolov8n-pose.pt format=onnx dynamic simplify`.
+
```

Comment:
Let's also add a standard `Performance Measurement` section to the README, i.e. download data -> run script

Reply:
done